src/main/java/net/neoforged/neoforge/transfer/storage/ItemHelper.java

```diff
@@ -0,0 +1,96 @@
+package net.neoforged.neoforge.transfer.storage;
+
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import net.neoforged.neoforge.transfer.item.ItemResource;
+import net.neoforged.neoforge.transfer.transaction.Transaction;
+
+public class ItemHelper {
+    // Just add up to 10 apples to the inventory. Return how many were inserted.
+
+    public static int addApples(Storage<ItemResource> storage) {
+        var apple = ItemResource.of(Items.APPLE);
+        try (var tx = Transaction.openOuter()) {
+            int inserted = storage.insert(apple, 10, tx);
+            tx.commit();
+            return inserted;
```

Comment:
Would it be possible to have a helper default method that we annotate and document not to override that would basically allow this call to become:
```java
return storage.insertAndCommit(apple, 10, tx);
```

Reply:
I get that this would make the code shorter but conceptually it doesn't make sense to me. The transaction should be committed/closed at the same level as it's opened. However we could have a helper that will open a tx + insert + commit the tx I think.